ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/LedgerDB/OrphanArbitrary.hs

```diff
@@ -0,0 +1,35 @@
+{-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+module Test.Ouroboros.Storage.LedgerDB.OrphanArbitrary where
+
+import           Test.QuickCheck
+import           Data.Time.Clock (DiffTime, secondsToDiffTime, diffTimeToPicoseconds)
+import           Ouroboros.Consensus.Config.SecurityParam (SecurityParam(..))
+import           Ouroboros.Consensus.Storage.LedgerDB.DiskPolicy (RequestedSnapshotInterval(..))
+
+{-------------------------------------------------------------------------------
+  Orphan Arbitrary instances
+-------------------------------------------------------------------------------}
+
+instance Arbitrary SecurityParam where
+  arbitrary = SecurityParam <$> choose (0, 6)
+  shrink (SecurityParam k) = SecurityParam <$> shrink k
+
+instance Arbitrary RequestedSnapshotInterval where
+  arbitrary =
+    oneof [ requestedSnapshotInterval
```

Comment:
I suggest using `frequency` to make the `defaultSnapshotInterval` case much less likely. EG 9:1. We'd rather spend our "budget" on the case that has more parameters to explore.